packages/common/infra/src/livedata/index.ts

```diff
@@ -0,0 +1,293 @@
+import { DebugLogger } from '@affine/debug';
+import {
+  distinctUntilChanged,
+  EMPTY,
+  filter,
+  type InteropObservable,
+  map,
+  Observable,
+  type Observer,
+  of,
+  type OperatorFunction,
+  scan,
+  skip,
+  type Subscription,
+  switchMap,
+} from 'rxjs';
+import { BehaviorSubject, Subject } from 'rxjs';
```

Comment:
clean the imports, better to move type imports to a separated one?

Reply:
We need to add a lint rule to check, there are too many such codes